app/main.py

```diff
@@ -1,8 +1,39 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self, comfort_class: int, clean_mark: int
+                 , brand: str) -> None:
```

Comment:
```suggestion
    def __init__(self, comfort_class: int, clean_mark: int,
                 brand: str) -> None:
```
Comma must be in previous line